tests/tests.py

```diff
@@ -137,6 +135,68 @@ def test_insert_field_add_import():
         "test-resources/java/insert_field_and_initializer/", output_summaries
     )
 
+def test_delete_method_from_interface():
+    delete_method = Rule (
+        name= "delete_method",
+        query=
+        """(
+            ((method_declaration 
+                name: (_) @id) @method)
+            (#eq? @id "foo")
+        )
+        """,
+        replace_node="method",
+        replace="",
+        filters= set([
+            Filter(
+                enclosing_node= "(class_declaration ) @c_cd",
+                contains = ["""(
+                    (super_interfaces (type_list (type_identifier) @it)
+                    (#eq? @it "FooBar"))
+                )""",],
+            )
+        ]),
+    )
+
```

Comment:
This rule is not right because it will delete all interfaces. But I didn't figure out a way of expressing that with the query language? Maybe we need two rules here for the case we have just one interface vs multiple?

Reply:
hmmm. I think this is a slight mis-use of the contains. I would rather just match `FooBar` it in the `enclosed_node`. I kinda see what you are trying to do. This is actually a limitation currently, its a bit awkward to delete a child node from a "sequence". If at all. were to do this I would - `(_)* (type_identifier) @it (_)*`. 

How about u update this example to delete unused private method or field or local var  ?